OpenRobertaRobot/src/main/java/de/fhg/iais/roberta/syntax/lang/expr/NullConst.java

```diff
@@ -41,42 +39,4 @@ public Object getValue() {
         return null;
```

Comment:
This is probably the most useless method, I have ever seen 🤣 . Please remove it (I know it's not you fault)